README.md

```diff
@@ -57,10 +57,22 @@ function queryCallback () {
 ## API
 
 ```js
-var getGeoJSON = require('osm-p2p-geojson')
+var osmGeoJson = require('osm-p2p-geojson')
 ```
 
-### var stream = getGeoJSON(osm[, options][, callback])
+### var importer = osmGeoJson.importer(osm)
+
+Create an importer for importing GeoJSON objects into the given osm-p2p database. Can track progress through listening tot he 'import' event. This event gives you the index of the most recently imported data and how many documents will be imported in total.
+
+```js
+var importer = osmGeoJson.importer(osm)
+importer.on('import', function (index, total) {
+  console.log('import', index, total)
+})
+importer.importFeatureCollection(geojson, cb)
```

Comment:
Looking at the code, it seems like you might get weird behavior if you called `importFeatureCollection` twice: the event emitter would emit `import` events for two separate import tracks. I wonder if the API could be structured differently to avoid this. Like, maybe just an  `import(geojson, cb)` function that returned an event emitter? I'm not sure..

Reply:
Ok this sounds good